template/chrome.manifest.mustache

```diff
@@ -1 +1 @@
-resource {{addon.chrome}} .
+content {{addon.chrome}} .
```

Comment:
Please add a newline at the end of this file.

Reply:
Done